gradle.properties

```diff
@@ -116,6 +116,7 @@ artifactRedirecting.androidx.annotation.version=1.9.1
 artifactRedirecting.androidx.graphics.version=1.0.1
 artifactRedirecting.androidx.lifecycle.version=2.9.0-alpha03
 artifactRedirecting.androidx.navigation.version=2.8.4
+artifactRedirecting.androidx.performance.version=1.0.0-alpha01
```

Comment:
Do we know if it will become stable before 1.8.0 Stabe release?

Reply:
If not, Jetpack Compose 1.8 has the same issue, and they also will stabilize it, or will revert it in the release

Reply:
yep, it should be aligned with Google. They won't have 1.8 release without stabilizing it